Cloud users are able to request the audit log (#4106)
circleci/circleci-docs: jekyll/_cci2/security.md

{% endraw %}

## Audit Logs

> CircleCI Server customers can access the audit log feature from UI.

Reviewer: ```suggestion
CircleCI Server customers can access the audit log feature from the UI.
```

Reviewer: Sorry I missed a word out the first time! Must have done that before I had any coffee :-)

Author: NP, Thank you for your a lot of help 🎉

> Server customers who installed CircleCI on your environments are able to use the audit log feature on UI.

Reviewer: ```suggestion
CircleCI Server customers can access the audit log feature from UI.
```

> ※ Cloud customer needs to [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request Audit log, and only an admin user of your organization is able to request this.

Reviewer: ```suggestion
Cloud customers can [contact CircleCI support](https://support.circleci.com/hc/en-us/requests/new) to request an Audit log. **Note:** only organization admin users can make an audit log request.
```
